### ISSUE TITLE
Blindness accessibility

### ISSUE THREAD

**Issue by AKArien** [smogon/pokemon-showdown#12030]
I'm not very familiar with screen reader solutions and accessibility myself, but i saw [this reddit post](https://www.reddit.com/r/pokemonshowdown/s/Bl7DzkJg9Q) of a would-be blind user turned away by the lack of accessibility for their disability. More accessibility is always nice !

**Comment by AKArien** [comment]
Oh, oops, hadn’t realised this was the server only, and that [such an issue](https://github.com/smogon/pokemon-showdown-client/issues/954) is already oppened there.
